kinetic/distribution.yaml

```diff
@@ -10259,6 +10259,16 @@ repositories:
       url: https://github.com/PX4/px4_msgs-release.git
       version: 1.0.0-3
     status: developed
+  py_script-release:
```

Comment:
While this name isn't used for package names, it should still be descriptive of what the package does.  In this case, it looks like this wants to be something more like `humanoid_shp`, or something like that.